deploy/kubernetes/deploy-kiali-to-kubernetes.sh

```diff
@@ -228,3 +228,23 @@ do
     exit 1
   fi
 done
+
+# Deploy Kiali MonitoringDashboards to kubernetes
+# Note for undeploy script: dashboards are implicitely undeployed when the related CRD is removed
+echo "Deploying Kiali dashboards to Kubernetes project ${NAMESPACE}"
+for dashboard in nodejs thorntail vertx-client vertx-eventbus vertx-pool vertx-server
+do
+  yaml_path="${YAML_DIR}/../dashboards/${dashboard}.yaml"
+  if [ -f "${yaml_path}" ]; then
+    echo "Using YAML file: ${yaml_path}"
+    cat ${yaml_path} | envsubst | kubectl apply -n ${NAMESPACE} -f -
+  else
+    get_downloader
+    yaml_url="https://raw.githubusercontent.com/kiali/kiali/${VERSION_LABEL}/deploy/dashboards/${dashboard}.yaml"
+    echo "Downloading YAML via: ${downloader} ${yaml_url}"
+    ${downloader} ${yaml_url} | envsubst | kubectl apply -n ${NAMESPACE} -f -
+  fi
+  if [ "$?" != "0" ]; then
+    echo "WARNING: Failed to deploy runtimes dashboards. If you need them, please deploy manually."
```

Comment:
Is there anything else we can say here to indicate this isn't really a problem that will break Kiali, that this is an optional feature. I see it is WARNING not ERROR. But I just have a feeling we need to be more explicit to tell the user something to the effect of "you don't really need these to run the base Kiali ...these are just for the optional dashboards." I don't know a good way to say it, but you know what I mean. :)

Reply:
How about: "WARNING: Failed to deploy runtimes dashboards. They are not mandatory and won't prevent Kiali to work. If you want to monitor your application runtimes, you can still deploy dashboards manually."

Ideally I would add a link to the documentation, but I don't have it yet